exonum/src/storage/keys.rs

```diff
@@ -24,6 +26,13 @@ use crypto::{Hash, PublicKey, HASH_SIZE, PUBLIC_KEY_LENGTH};
 /// will not sort signed integer types in the natural order; therefore, they are
 /// mapped to the corresponding unsigned type by adding a constant to the source value.
 ///
+/// The `SystemTime` (de)serialization is only possible for the values greater than
+/// 1970-01-01 00:00:00 UTC. Since the `SystemTime` type saves time in the form
+/// `(seconds: u64, nanoseconds: u32)`, the total occupied size of `SystemTime` in the storage
+/// is 12 bytes, the time is stored as a buffer of bytes `buffer[0..8]: &[u8]`.
```

Comment:
I don't quite get the meaning of "the time is stored..." part; imo, it could be safely removed.